src/app/pages/common/entity/entity-form/components/form-explorer/form-explorer.component.ts

```diff
@@ -89,63 +91,31 @@ export class FormExplorerComponent implements Field, OnInit {
       this.customTemplateStringOptions = this.config.customTemplateStringOptions;
       this.config.customTemplateStringOptions.explorer = this;
     }
-    if (this.config.explorerType === 'zvol') {
-      this.displayFieldName = 'name';
-      this.nodes = [{
-        mountpoint: this.config.initial,
-        name: this.config.initial,
-        hasChildren: true,
-        expanded: !this.rootSelectable,
-      }];
-    } else {
-      this.displayFieldName = 'subTitle';
-      this.nodes = [{
-        name: this.config.initial,
-        subTitle: this.config.initial,
-        hasChildren: true,
-        expanded: !this.rootSelectable,
-      }];
-    }
+
+    this.displayFieldName = 'subTitle';
+    this.nodes = [{
+      name: this.config.initial,
+      subTitle: this.config.initial,
+      hasChildren: true,
+      expanded: !this.rootSelectable,
+    }];
 
     this.customTemplateStringOptions.displayField = this.displayFieldName;
```

Comment:
`this.displayFieldName` only used once. Remove `displayFieldName` and set value directly `this.customTemplateStringOptions.displayField = 'subTitle'`